src/__internal__/validation-message/validation-message.spec.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { mount } from "enzyme";
+import ValidationMessage from ".";
+import StyledValidationMessage from "./validation-message.style";
+import { assertStyleMatch } from "../../__spec_helper__/test-utils";
+import { baseTheme } from "../../style/themes";
+
+const render = (props) => mount(<ValidationMessage {...props} />);
+
+const validationWithStrings = [
+  ["error", "warning"],
+  [undefined, "warning"],
+];
+const validationWithBooleans = [
+  [true, "string"],
+  [undefined, true],
+];
+
+const computeId = (id) => (id === "warning" ? "warningText" : id);
+
+const addColorId = (arr) =>
+  arr.map((subArr) => [...subArr, computeId(subArr[0] || subArr[1])]);
+
+describe("ValidationMessage component", () => {
+  let wrapper;
+
+  it.each(addColorId(validationWithStrings))(
```

Comment:
Some of these are rendering `%s` in the description: 

![image](https://user-images.githubusercontent.com/14963680/142236722-9c5219fc-34fd-4444-b553-9466bd75e3d2.png)

Reply:
Yeah I originally had info but removed when I clarified with DS, I'll get it sorted